build.sbt

```diff
@@ -23,6 +23,7 @@ import spray.boilerplate.BoilerplatePlugin
 initialize := {
   // Load system properties from a file to make configuration from Jenkins easier
   loadSystemProperties("project/akka-build.properties")
+  assert(CrossJava.Keys.fullJavaHomes.value.contains("8"), "JDK 8 is not installed but required to build akka")
```

Comment:
I think we have a better check for that
```scala
JavaVersion.isJdk8
```

Reply:
As I understood this only returns true if system jdk (used to build akka) is JDK 8, not if JDK 8 is installed and found.

Reply:
Though I do not think we want to assert Jdk 8 only.

Reply:
https://github.com/akka/akka/blob/31f6170c623867523eeca25c5626f006bee727cf/project/AkkaBuild.scala#L122 forces JDK 8 if system default is not JDK 8 (without checking if JDK 8 is installed/found)